sdk/tests/js/webgl-test-utils.js

```diff
@@ -1688,7 +1668,35 @@ var glErrorShouldBe = function(gl, glErrors, opt_msg) {
   glErrorShouldBeImpl(gl, glErrors, true, opt_msg);
 };
 
+/**
+ * Tests that the first error GL returns is in the specified error list.
+ * @param {!WebGLContext} gl The WebGLContext to use.
+ * @param {!Array.<number>} expectedErrorList The list of expected gl errors.
+ * @param {string} opt_msg
+ */
+var glErrorShouldBeIn = function(gl, expectedErrorList, opt_msg) {
```

Comment:
Why was this needed? It looks like it has the same basic functionality as glErrorShouldBeImpl, below.